client-datastore-rdbms/src/main/resources/schema-postgresql.sql

```diff
@@ -41,4 +42,6 @@ create table if not exists client_session (
 
 create index if not exists idx_clientSession_customerId on client_session (customerId);
 create index if not exists idx_clientSession_locationId on client_session (customerId, locationId);
+create index if not exists idx_clientSession_customerId_macAddressString on client_session (customerId, macAddressString);
```

Comment:
actually, this statement will fail during the upgrade - because the table already exists, but the column macAddressString does not exist.